integration-tests/src/main/java/org/infinispan/quarkus/embedded/TestServlet.java

```diff
@@ -143,4 +147,30 @@ public String create(@PathParam("id") String id, Book book) {
         books.put(id, book);
         return id;
     }
+
+    @Path("ANNOTATIONS/GET/review/{bookId}")
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    public String create(@PathParam("bookId") String bookId) {
+        Objects.requireNonNull(bookId);
+        return reviewService.getReview(bookId);
+    }
+
+    @Path("ANNOTATIONS/GET/clear/{bookId}")
+    @GET
+    public void clear(@PathParam("bookId") String bookId) {
+        Objects.requireNonNull(bookId);
+        if (bookId.equals("all")) {
```

Comment:
This sounds like a different operation to me. I guess it is fine though as it is a test.